classes/output/questions_table.php

```diff
@@ -143,8 +143,24 @@ protected function define_table_columns() {
      */
     protected function define_table_configs() {
         $this->collapsible(false);
-        $this->sortable(true, get_string('new', 'local_qtracker'), SORT_DESC);
+        $this->sortable(true);
         $this->pageable(true);
+        $sortdata = [
+            [
+                'sortby' => 'new',
+                'sortorder' => SORT_DESC,
+            ], [
+                'sortby' => 'open',
+                'sortorder' => SORT_DESC,
+            ], [
+                'sortby' => 'closed',
+                'sortorder' => SORT_DESC,
+            ], [
+                'sortby' => 'id',
+                'sortorder' => SORT_ASC,
```

Comment:
@davrikn Shouldn't the **id** also be sorted by `SORT_DESC`? I think we want the questions with the highest id (newest) listed first.